Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/EmojiPickerViewController.swift

```diff
@@ -93,13 +144,10 @@ extension EmojiPickerViewController: UITableViewDelegate, UITableViewDataSource
         defer {
             tableView.deselectRow(at: indexPath, animated: true)
         }
-        guard let packageId = packages[indexPath.row].packageID else { return }
-        downloadedPackage.append(packageId)
-        tableView.reloadRows(at: [indexPath], with: .automatic)
-        if packages[indexPath.row].isDownload != "Y" {
-            StickerApiClient.downloadStickers(packageId: packages[indexPath.row].packageID ?? 0) { _ in }
-        } else {
-            StickerApiClient.hideStickers(packageId: packages[indexPath.row].packageID ?? 0, nil)
+        if let sendEmojiVc: SendEmojiViewController = SendEmojiViewController.instantiateController(storyboard: .wallet) {
+            sendEmojiVc.packageInfo = packages[indexPath.row]
+            sendEmojiVc.chatChannelController = chatChannelController
+            self.present(sendEmojiVc, animated: true)
```

Comment:
@mdhanif-simformsolutions  `self` not required here.

Reply:
Updated this thanks.